patches/octane_run_we.js

```diff
@@ -0,0 +1,47 @@
+load('base.js');
+load('richards.js');
+load('deltablue.js');
+load('crypto.js');
+load('raytrace.js');
+load('earley-boyer.js');
+load('regexp.js');
+load('splay.js');
+load('navier-stokes.js');
+load('pdfjs.js');
+//load('mandreel.js');
+//load('gbemu-part1.js');
+//load('gbemu-part2.js');
+//load('code-load.js');
+//load('box2d.js');
+//load('zlib.js');
+//load('zlib-data.js');
+load('typescript.js');
+load('typescript-input.js');
+load('typescript-compiler.js');
+
+
+function main() {
+  var suites = BenchmarkSuite.suites;
+  for (var i = 0; i < suites.length; i++) {
```

Comment:
I was about to suggest that if `i` is only used to fetch the right array index out, use a foreach, but it seems javascript doesn't have one. Or one that everyone can agree on, anyway...